WebDriverAgentLib/Categories/XCUIElement+FBScrolling.m

```diff
@@ -80,9 +80,9 @@ - (BOOL)fb_scrollToVisibleWithNormalizedScrollDistance:(CGFloat)normalizedScroll
   if (self.fb_isVisible) {
     return YES;
   }
-  XCElementSnapshot *scrollView = [self.lastSnapshot fb_parentMatchingType:XCUIElementTypeScrollView];
+  XCElementSnapshot *scrollView = [self.lastSnapshot fb_parentMatchingType:XCUIElementTypeCollectionView];
   scrollView = scrollView ?: [self.lastSnapshot fb_parentMatchingType:XCUIElementTypeTable];
-  scrollView = scrollView ?: [self.lastSnapshot fb_parentMatchingType:XCUIElementTypeCollectionView];
+  scrollView = scrollView ?: [self.lastSnapshot fb_parentMatchingType:XCUIElementTypeScrollView];
```

Comment:
Yes you are right. I read your [comment](https://github.com/facebook/WebDriverAgent/issues/228#issuecomment-237214008) . Let me try to make this change. Because the current one only works if parent element is scrollview.